kloppy/infra/serializers/event/opta/deserializer.py

```diff
@@ -310,6 +317,36 @@ def _parse_shot(
     return dict(coordinates=coordinates, result=result, qualifiers=qualifiers)
 
 
+def _parse_duel(raw_qualifiers: List, type_id: int, outcome: int) -> Dict:
+    qualifiers = _get_event_qualifiers(raw_qualifiers)
+    if type_id == EVENT_TYPE_TACKLE:
+        qualifiers.extend([DuelQualifier(value=DuelType.GROUND)])
+    elif type_id == EVENT_TYPE_AERIAL:
+        qualifiers.extend(
+            [
+                DuelQualifier(value=DuelType.LOOSE_BALL),
+                DuelQualifier(value=DuelType.AERIAL),
+            ]
+        )
+    elif type_id == EVENT_TYPE_50_50:
+        qualifiers.extend(
+            [
+                DuelQualifier(value=DuelType.LOOSE_BALL),
+                DuelQualifier(value=DuelType.GROUND),
+            ]
+        )
+
+    if outcome:
+        result = DuelResult.WON
+    else:
+        result = DuelResult.LOST
+
+    return dict(
+        result=result,
```

Comment:
This can be simplified to `result=DuelResult.WON if outcome else DuelResult.LOST`